src/dbSta/test/hierclock_out.vok

```diff
@@ -200,7 +200,7 @@ module counter (clk_i,
     .Q(counter_q[3]),
     .QN(\U2/_38_ ));
 endmodule
-module counter-1 (clk_i,
+module counter.U3 (clk_i,
```

Comment:
This looks like a weird verilog name to me, why create a module called counter.U3 ?
This might be related to the createDbModule taking in an instance name. It is ok to simply have a unique module name without worrying about the instance names (in a uniquified hierarchy, all cells have unique names).

Reply:
read_verilog fails with module names like counter-1.

Reply:
ok, try _1.  Internally Yosys can do this*
The "." catches my attention because Verilog has something called xmr's (external module references eg U1.U2.U3 is a hierarchical name which could be accessed globally). The "." has special meaning.


*note how yosys uniquify strings:
RTLIL::IdString RTLIL::Module::uniquify(RTLIL::IdString name, int &index)
{
        if (index == 0) {
                if (count_id(name) == 0)
                        return name;
                index++;
        }

        while (1) {
                RTLIL::IdString new_name = stringf("%s_%d", name.c_str(), index);
                if (count_id(new_name) == 0)
                        return new_name;
                index++;
        }
}